meta-lmp-base/recipes-security/optee/optee-os_git.bb

```diff
@@ -1,32 +1,16 @@
 SUMMARY = "OP-TEE Trusted OS"
+HOMEPAGE = "https://github.com/foundries.io/optee_os.git" 
```

Comment:
Looks like there's extra white-space on the end of this line.

Reply:
Address is also incorrect, should be "https://github.com/foundriesio/optee_os"

Reply:
yes

Reply:
done